### DISCUSSION TITLE
Include python notebooks in search results

### DISCUSSION THREAD

**Discussion by nside** [community/community#39873] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I noticed Python notebooks aren't included in search results. Please index .ipynb documents

**Comment by ossanna16** [comment]
Hi there @nside and welcome to our community! Thank you for the feedback!

**Comment by look** [comment]
@nside There are no special restrictions on `.ipynb` documents ([example search](https://github.com/search?q=path%3A*.ipynb&type=code)).

However, due to their nature, many of these notebooks are quite large. You may not be able to find your content due to [the 350 KiB file limit](https://docs.github.com/en/search-github/github-code-search/about-github-code-search#limitations). Please check your document size and let us know how big they are so we can collect data about this limitation.
